declaration.d.ts

```diff
@@ -39,6 +41,9 @@ declare global {
       WALLET_COUNT: number;
     }
   }
+
+  var environment: Environment;
```

Comment:
Is this one diferent from the environment property already in the daedalus object?

Reply:
The same change is already in develop, when I updated the branch the diff shows no changes to `declaration.d.ts`